utils/airtable_es2015.js

```diff
@@ -42,14 +42,19 @@ var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateF
     dataStore[tableName] = await fetchTableFromAirtable(tableName);
   });
   await Promise.all(promises);
-
+  dataStore["errors"] = [];
   airtableConstants.tableNames.forEach(function(tableName) {
     var array = dataStore[tableName].map(x => Object.keys(x).length);
     var number_of_fields = Math.max(...array);
     dataStore[tableName] = dataStore[tableName].filter(x => {
       var fraction_of_cols_filled =
         (Object.keys(x).length * 1) / number_of_fields;
-      return fraction_of_cols_filled < 0.5 ? false : true;
+      if (fraction_of_cols_filled < 0.5) {
+        dataStore["errors"].push("missingValues." + tableName);
```

Comment:
If there are multiple rows with missing values then this message will be duplicated in `errors` - is that the intent?

Reply:
Good point, I just added the row number to the error message